src/System.Data.OleDb/tests/Helpers.cs

```diff
@@ -29,13 +31,15 @@ static Nested()
                 List<object> providerNames = new List<object>();
                 foreach (DataRow row in table.Rows)
                 {
-                    providerNames.Add(row[providersRegistered]);
+                    providerNames.Add((string)row[providersRegistered]);
                 }
-                string providerName = PlatformDetection.Is32BitProcess ? 
-                    @"Microsoft.Jet.OLEDB.4.0" : 
-                    @"Microsoft.ACE.OLEDB.12.0";
-                IsAvailable = false; // ActiveIssue #37823 // providerNames.Contains(providerName);
-                ProviderName = IsAvailable ? providerName : null;
+                // skip if x86 or if the expected driver not available 
+                IsAvailable = !PlatformDetection.Is32BitProcess && providerNames.Contains(ExpectedProviderName);
+                if (CultureInfo.CurrentCulture.Name.Equals("es-ES", StringComparison.OrdinalIgnoreCase))
```

Comment:
What do you think about changing this detection to say that run the tests only on en-US and no other locale ?

Reply:
IN case the CI incorporates other cultures, then those tests may start failing because the input file was not from the same locale.

Reply:
You're right! I'll make that change